crates/common/tedge_config/Cargo.toml

```diff
@@ -9,11 +9,14 @@ homepage = { workspace = true }
 repository = { workspace = true }
 
 [dependencies]
+camino = { version = "1.1.4", features = ["serde", "serde1"] }
 certificate = { path = "../certificate" }
+doku = "0.20.0"
 figment = { version = "0.10", features = ["env", "toml"] }
+gusket = "0.1.0"
```

Comment:
Do we have so many getter/setter to benefit from this new dependency?

Reply:
I don't think we do, I'll remove it as I was just testing something and not using it anywhere